.eslintrc

```diff
@@ -1,12 +1,27 @@
 ---
 parser: babel-eslint
 
+plugins:
+  - react
+
 env:
   browser: true
   node: true
 
 globals:
   __DEV__: true
+  # Jest / Jasmine
+  describe: false
```

Comment:
Any difference between true and false here? If not let's be consistent (I don't care what the value is if it works)